app/views/directives/add-secret-to-application.html

```diff
@@ -0,0 +1,90 @@
+<div>
+  <div class="dialog-title">
+    <h3>Add to Application</h3>
+  </div>
+  <div class="dialog-body">
+    <form name="addToApplicationForm" novalidate>
+      <fieldset ng-disabled="disableInputs">
+        <div class="form-group">
+          <label>Add this secret to application:</label>
```

Comment:
Ideally label elements get a for attribute to associate them with the input they are for (here and throughout the file).